src/picknik_ur_site_config/objectives/get_lever_handle_graspable_object.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<root BTCPP_format="4" main_tree_to_execute="GetLeverHandleGraspableObject">
+  <!-- ////////// -->
+  <!-- TODO: need to be directly in front of handle for this behavior to work until (https://github.com/PickNikRobotics/moveit_studio/issues/5968) is fixed or mesh completion behaviors are completed. -->
+  <BehaviorTree ID="GetLeverHandleGraspableObject" _favorite="true">
```

Comment:
For consistency, call it "Get Lever Handle Graspable Object" with spaces, also no need to link to private issues here either.